modules/calib3d/include/opencv2/calib3d.hpp

```diff
@@ -526,6 +532,25 @@ enum HandEyeCalibrationMethod
     CALIB_HAND_EYE_DANIILIDIS   = 4  //!< Hand-Eye Calibration Using Dual Quaternions @cite Daniilidis98
 };
 
+enum SamplingMethod { Uniform, ProgressiveNAPSAC, Napsac, Prosac, Evsac };
+enum LocalOptimMethod {NullLO, InLORsc, InItFLORsc, GC, SIGMA};
+enum ScoreMethod {usacRANSAC, usacMSAC, usacMAGSAC, usacLMeds};
+enum NeighborSearchMethod { Flann, Grid, RadiusSearch };
+
+struct CV_EXPORTS_W_SIMPLE UsacParams
+{ // in alphabetical order
+    double confidence = 0.99;
+    bool is_parallel = false;
```

Comment:
for public structures we use mixed case notation, `is_parallel` => `isParallel` etc.

Reply:
done